core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -79,6 +79,7 @@ export class Exploration extends BaseTranslatableObject {
   states: States;
   title: string;
   languageCode: string;
+  metadata: ExplorationMetadataBackendDict;
```

Comment:
The Exploration object in the backend does not have a metadata dict. Why are you introducing one here?

If this is necessary for the frontend then you'd need to call this metadataDict and convert it to a domain object as well.